tools/capacity-planner-sheets-extension/queries.js

```diff
@@ -0,0 +1,136 @@
+/**
+ * Copyright 2023 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * 
+ * This file stores the queries for the Google Cloud Monitoring API.
+ * 
+ * NOTE: Apps Script only allows HTML or .gs files so there is no simple way to include a TOML or JSON file here.
+  Queries are originally from the queries.toml file below.
+
+  [TODO: link to Capacity Planner CLI TOML file]
+*/
+
+const QUERIES = {
+  "bigtable": {
+    "product_name": "Cloud BigTable",
+    "metrics": {
+      "egress": {
+        "metric_name": "Egress MB/s",
+        "query": "fetch bigtable_table\n        | metric 'bigtable.googleapis.com/server/sent_bytes_count'\n        | align rate(1m)\n        | every 1m\n        | group_by [], [value_sent_bytes_count_aggregate: aggregate(value.sent_bytes_count)]\n        | scale ('MiBy/s')"
```

Comment:
I noticed there is a lot of extra spacing in this file such as this line and other lines. Is this spacing needed or was it create by mistake?

Reply:
@shunjikawabata Do you know if the monitoring API cares about the extra spaces?

Reply:
It looks like the spaces are from the TOML (where the spaces are actually line breaks). The conversion script strips away the extra spaces now.